support-workers/src/main/scala/com/gu/acquisitions/AcquisitionDataRowBuilder.scala

```diff
@@ -0,0 +1,215 @@
+        case (Collection, Weekend) => VoucherWeekend
+        case (Collection, WeekendPlus) => VoucherWeekendPlus
+        case (Collection, Saturday) => VoucherSaturday
+        case (Collection, SaturdayPlus) => VoucherSaturdayPlus
+        case (Collection, Sunday) => VoucherSunday
+        case _ => VoucherSundayPlus
+      }
+
+    product match {
+      case p: Paper => Some(PrintOptions(
+       printProduct(p.fulfilmentOptions, p.productOptions),
+        Country.UK
+      ))
+      case _: GuardianWeekly => Some(PrintOptions(
+        PrintProduct.GuardianWeekly,
+        deliveryCountry.getOrElse(Country.UK)
+      ))
+      case _ => None
+    }
+  }
+
+  private def getAcquisitionTypeDetails(s: SendThankYouEmailState): AcquisitionTypeDetails =
+    s match {
+      case s: SendThankYouEmailContributionState => AcquisitionTypeDetails(
+        Some(s.paymentMethod),
+        None,
+        Direct,
+        Purchase,
+        Some(s.accountNumber),
+        Some(s.subscriptionNumber),
+      )
+      case s: SendThankYouEmailDigitalSubscriptionDirectPurchaseState => AcquisitionTypeDetails(
+        Some(s.paymentMethod),
+        s.promoCode,
+        Direct,
+        Purchase,
+        Some(s.accountNumber),
+        Some(s.subscriptionNumber)
+      )
+      case s: SendThankYouEmailDigitalSubscriptionGiftPurchaseState => AcquisitionTypeDetails(
+        Some(s.paymentMethod),
+        s.promoCode,
+        Gift,
+        Purchase,
+        Some(s.accountNumber),
+        Some(s.subscriptionNumber),
+      )
+      case s: SendThankYouEmailPaperState => AcquisitionTypeDetails(
+        Some(s.paymentMethod),
+        s.promoCode,
+        Direct,
+        Purchase,
+        Some(s.accountNumber),
+        Some(s.subscriptionNumber)
+      )
+      case s: SendThankYouEmailGuardianWeeklyState => AcquisitionTypeDetails(
+        Some(s.paymentMethod),
+        s.promoCode,
+        if (s.giftRecipient.isDefined) Gift else Direct,
+        Purchase,
+        Some(s.accountNumber),
+        Some(s.subscriptionNumber)
+      )
+      case s: SendThankYouEmailDigitalSubscriptionCorporateRedemptionState => AcquisitionTypeDetails(
+        None,
+        None,
+        Corporate,
+        Redemption,
+        Some(s.accountNumber),
+        Some(s.subscriptionNumber)
+      )
+      case s: SendThankYouEmailDigitalSubscriptionGiftRedemptionState => AcquisitionTypeDetails(
+        None,
+        None,
+        Gift,
+        Redemption,
+        None, //TODO: if we rework digital gift modelling in Zuora we should include the relevant ids here
+        None
+      )
+    }
+
+  private def buildLabels(state: SendAcquisitionEventState, accountExists: Boolean) =
+    Set(
+      if (accountExists) Some("REUSED_EXISTING_PAYMENT_METHOD") else None,
+      if (isSixForSix(state)) Some("GUARDIAN_WEEKLY_SIX_FOR_SIX") else None,
+      if (state.analyticsInfo.isGiftPurchase) Some("GIFT_SUBSCRIPTION") else None,
```

Comment:
changing this from gift-subscription to GIFT_SUBSCRIPTION broke the flagging in GA of gift subs because it relied on this string https://github.com/guardian/acquisition-event-producer/pull/50/files#diff-eadf20916668eea4ad74100913282140802379c1d17a35411e24832732b7868bR89